src/main/java/io/github/yeffycodegit/poglog/PogLogger.java

```diff
@@ -0,0 +1,83 @@
+package io.github.yeffycodegit.poglog;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Arrays;
+
+public class PogLogger {
+    private final boolean logToFile;
+    private String filename;
+    FileWriter fileWriter;
+
+    // Define color constants
+    public static final String RESET = "\u001B[0m";
+    public static final String BLACK = "\u001B[30m";
+    public static final String RED = "\u001B[31m";
+    public static final String GREEN = "\u001B[32m";
+    public static final String YELLOW = "\u001B[33m";
+    public static final String BLUE = "\u001B[34m";
+    public static final String PURPLE = "\u001B[35m";
+    public static final String CYAN = "\u001B[36m";
+    public static final String WHITE = "\u001B[37m";
+
+    public PogLogger(boolean logToFile) throws IOException {
+        this.logToFile = logToFile;
+        this.filename = "app-logs.txt";
+
+        fileWriter = new FileWriter(filename);
+    }
+
+    public void success(Object msg) throws IOException {
+        if (logToFile) {
+            fileWriter.write(GREEN + "[SUCCESS] " + msg.toString() + RESET + "\n");
```

Comment:
This is writing to a file, not a console. You can't have colours in a text file, this will just write some strange looking text.